starport/cmd/scaffold_module.go

```diff
@@ -20,6 +20,7 @@ const (
 	flagIBC                 = "ibc"
 	flagParams              = "params"
 	flagIBCOrdering         = "ordering"
+	flagNoRegistration      = "no-registration"
```

Comment:
@Pantani sorry but idk where to write the logic for it and how to do so , you said we already doing so when there is a error with ` app`, can you share the link where its written, Thanks :D

Reply:
@championshuttler, you should only parse the flag into the `cmd` and handle the logic into the [`app.go` modifier](https://github.com/tendermint/starport/blob/master/starport/templates/module/create/stargate.go#L74-L82)